mpas_analysis/shared/io/mpas_reader.py

```diff
@@ -194,7 +194,10 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
         # convert to string
         timeStrings = [''.join(xtime.astype('U')).strip()
                        for xtime in timeVar.values]
-        days = string_to_days_since_date(dateString=timeStrings,
+        for i, timeString in enumerate(timeStrings):
+             if timeString[8:] == '15_00:00:00':
+                 timeStrings[i] = f'{timeStrings[i+1][:7]}-01_00:00:00'
+             days = string_to_days_since_date(dateString=timeStrings,
```

Comment:
This didn't work for me.  Should this be:
```suggestion
        for i, timeString in enumerate(timeStrings):
             if timeString[8:] == '15_00:00:00':
                 timeStrings[i] = f'{timeStrings[i+1][:7]}-01_00:00:00'
        days = string_to_days_since_date(dateString=timeStrings,
```